warehouse/warehouse.go

```diff
@@ -457,56 +562,60 @@ func (wh *HandleT) createLoadFiles(job *ProcessStagingFilesJobT) (err error) {
 
 	wg := misc.NewWaitGroup()
```

Comment:
We are not using this waitGroup anymore?

Reply:
removed